articles/is-node-js-a good-choice-to-create-an-app-based-on-micro-services/index.md

```diff
@@ -0,0 +1,36 @@
+
+In a monolithic architecture, tasks are not clearly divided nor defined. For instance, fixing a login issue in a huge application could require a developer to first understand the whole codebase of an application, instead of just one straightforward process.
+
+#### Benefits of micro-services
+
+The choice of technology when building applications is not important compared to the business rationale. Introducing micro-services into an app development project provides tangible benefits, as highlighted below.
+
+1. Better structure: a well-structured app is usually easier to understand, meaning that any new features to be added to the app will be easier to design and implement. And therefore, less time and cost is spent on business analysis. Thus developers spend more time on development.
+2. Seamless implementation of complex products: complex applications are difficult to conceptualize, build, and maintain. But with micro-services, they are broken down into small bits that are more manageable.
+3. Scalability: scaling a monolithic app with a huge [blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) of tightly coupled code is difficult, since a lot of time is spent trying to figure out what&#39;s going on inside. With micro-services, it becomes more apparent, which are the bottlenecks, and therefore getting them to handle more load is easier.
+4. Fewer errors: in a monolithic architecture looking for bugs is challenging and time consuming compared to micro-services where the small bit-sized pieces of technology are kept error-free and thus more manageable.
+5. Better performance: every micro-service only does one job at a time, meaning that they are easy to tune and optimize for work efficiency.
+6. Resistance to vendor lock-in: Offers flexibility as different programming languages can be used depending on the problem.
+
+### Understanding Node.js technology and its benefits
+
+Node.js is an event-driven [JavaScript runtime](http://dolszewski.com/javascript/javascript-runtime-environment/) designed to build applications that can be scaled afterward and runs on the web. Micro-service-based apps build in Node.js can deliver a lot of benefits, as highlighted below.
+
+1. Performance and reliability: the app will be stable and can be scaled to infinity depending on cost. One of the great features of Node.js that enables this is its single-threaded asynchronous architecture meaning that it [spawns](https://subscription.packtpub.com/book/application_development/9781785289583/3/ch03lvl1sec34/how-to-spawn-a-process) child processes for some asynchronous tasks, however these tasks are invisible to a programmer and run deep under the hood.
+2. Data replication: this is easy with Node.js and is also required by micro-services.
```

Comment:
You could expand on this and explain what data replication is for beginners who may not understand.